doc(release) Release notes oss 24.10 juil.26 (#5512)
centreon/centreon-documentation: versioned_docs/version-24.10/releases/centreon-os.mdx

| Centreon AWIE | [24.10.8 (July 22, 2026)](#centreon-awie-24108)<br/>[24.10.7 (June 24, 2026)](#centreon-awie-24107)<br/>[24.10.6 (May 4, 2026](#centreon-awie-24106)<br/>[24.10.5 (February 25, 2026)](#centreon-awie-24105)<br/>[24.10.4 (January 26, 2026)](#centreon-awie-24104)<br/>[24.10.3 (December 31, 2025)](#centreon-awie-24103)<br/>[24.10.2 (December 18, 2025)](#centreon-awie-24102)<br/>[24.10.1 (March 10, 2025)](#centreon-awie-24101) |
| Centreon High Availability | [24.10.2 (December 18, 2025)](#centreon-high-availability-24102)<br/>[24.10.1 (August 6, 2025)](#centreon-high-availability-24101)<br/>[24.10.0 (October 31, 2024)](#centreon-high-availability-24100) |

</details>

## July 22, 2026

### Centreon Web 24.10.29

<details open>
  <summary>Enhancements</summary>

- [resource status] performance has been drastically improved for many search use cases.
- [unattended] added support for php84, el10 and deb13.
- [upgrade] upgrades are now fully traced in a dedicated upgrade.log file, making migrations easier to troubleshoot.

> - [web] new logging system for centreon-web.

Reviewer: ```suggestion
- [Resource status] Performance has been drastically improved for many search use cases.
- [Upgrade] A dedicated upgrade.log file now fully traces upgrades, making them easier to troubleshoot.
- [Web] A new logging system has been implemented for centreon-web.
```

</details>

<details>
  <summary>Bug fixes</summary>

- [configuration] fixed an issue where the '_' character no longer was a valid character in address definitions.

> - [configuration] fixed stream connector (lua) parameters disappearing after deleting one of them in a broker configuration.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where the '_' character was no longer valid in address definitions.
- [Configuration] Fixed stream connector (lua) parameters disappearing after deleting one of them in a Broker configuration.
```

> - [dashboards] it is now possible for users to share a dashboard with:contact groups part of a common acl access group.individual contacts listed in the contact groups they belong to.

Reviewer: ```suggestion
- [Dashboards] It is now possible for users to share a dashboard with:
   * contact groups part of a common acl access group.
   * individual contacts listed in the contact groups they belong to.
```

> - [graphs] non-admin users can now export meta service performance graphs as csv.

Reviewer: ```suggestion
- [Graphs] Non-admin users can now export metaservice performance graphs as CSV.
```

> - [host discovery] importing discovered hosts now reliably completes and reports a genuine status instead of a false success.

Reviewer: ```suggestion
- [Host discovery] Importing discovered hosts now reliably completes and reports a genuine status instead of a false success.
```

> - [web] fixed an issue in the extension manager where typing in the search bar or using type/status filters had no effect.

Reviewer: ```suggestion
- [Web] Fixed an issue in the extension manager where typing in the search bar or using type/status filters had no effect.
```

</details>

### Centreon Open Tickets 24.10.14

<details>
  <summary>Bug fixes</summary>

> - [open tickets] fixed “insufficient privileges” error when opening a ticket from a custom view without notification rules access.

Reviewer: ```suggestion
- [Open tickets] Fixed “insufficient privileges” error obtained when opening a ticket from a custom view without notification rules access.
```

  <summary>Enhancements</summary>

- [Resource status] Performance has been drastically improved for many search use cases.
- [Upgrade] A dedicated upgrade.log file now fully traces upgrades, making them easier to troubleshoot.
- [Web] A new logging system has been implemented for centreon-web.

</details>

<details>
  <summary>Bug fixes</summary>

- [Configuration] Fixed an issue where the '_' character was no longer valid in address definitions.
- [Configuration] Fixed stream connector (lua) parameters disappearing after deleting one of them in a Broker configuration.
- [Dashboards] It is now possible for users to share a dashboard with:
   * contact groups part of a common acl access group.
   * individual contacts listed in the contact groups they belong to.
- [Graphs] Non-admin users can now export metaservice performance graphs as CSV.
- [Host discovery] Importing discovered hosts now reliably completes and reports a genuine status instead of a false success.
- [Web] Fixed an issue in the extension manager where typing in the search bar or using type/status filters had no effect.

</details>

### Centreon Open Tickets 24.10.14

<details>
  <summary>Bug fixes</summary>

> - [Open tickets] Fixed “insufficient privileges” error obtained when opening a ticket from a custom view without notification rules access.

Reviewer: ```suggestion
- [Open tickets] Fixed “insufficient privileges” error occurring when opening a ticket from a custom view without notification rules access.
```
